examples/hybrid/cli_options.jl

```diff
@@ -151,7 +151,7 @@ function parse_commandline()
         arg_type = Int
         default = 10
         "--nh_poly"
-        help = "Horizontal polynomial order"
+        help = "Horizontal polynomial degree. Note: The number of quadrature points in 1D within each horizontal element is then Nq = <--nh_poly> + 1 by defualt"
```

Comment:
```suggestion
        help = "Horizontal polynomial degree. Note: The number of quadrature points in 1D within each horizontal element is then Nq = <--nh_poly> + 1."
```
Suggest removing the `by default` comment since this is true following our definitions of those terms (Putting in the default statement would be fine if we present an example with our default setting of  --nh_poly = 3). (Since this is the outward facing interface and has no information about over-integration for instance)

Reply:
okay, done.